load-test/scripts/loadTest.js

```diff
@@ -24,14 +24,14 @@ import metricsReporter from './scannerLoadTestMetrics.js';
 const USE_CUSTOM_METRICS = false;
 
 // Change these options to increase the user goal or time to run the test.
-getTestOptions(
+export const options = getTestOptions(
```

Comment:
Removed this export in a previous PR, didn't realize it was necessary for K6 to pick up options.